### PR TITLE
Add `PayLoad` as the default plugin for mg400_node

### PR DESCRIPTION
**Pull request by ynyBonfennil**
This pull request adds `PayLoad` as the plugin for mg400_node.

I've fixed the name `PayLoad` to `Payload` in the previous pull request. However it seems like the correct command name is `PayLoad` (must be a typo by dobot company), so I've fixed the name back to it.

- https://github.com/Dobot-Arm/TCP-IP-Protocol/blob/master/README-EN.md#39-payload

### CODE REVIEW

File: mg400_interface/src/commander/dashboard_commander.cpp [hunk 1]
```diff
@@ -94,13 +94,13 @@ uint64_t DashboardCommander::robotMode() const
   return static_cast<uint64_t>(ResponseParser::takeInt(response.ret_val));
 }
 
-void DashboardCommander::payload(
+void DashboardCommander::pay_load(
```
Comment (m12watanabe1a) [inline mg400_interface/src/commander/dashboard_commander.cpp]: The name roles for the method is camel-case in our project so it is better to change it `payLoad`

**Review by m12watanabe1a — CHANGES_REQUESTED** [review]
Check the commnet above and please merge this after fixing it.

File: mg400_interface/src/commander/dashboard_commander.cpp (hunk 1 above)
Comment (ynyBonfennil) [reply mg400_interface/src/commander/dashboard_commander.cpp]: Thank you for pointing it out. I'll fix it.

**Review by m12watanabe1a — APPROVED** [review]
LGTM

**Pull request merged**